packages/maskbook/src/social-network-adaptor/twitter.com/ui-provider.ts

```diff
@@ -171,6 +173,8 @@ const twitterUI: SocialNetworkUI.Definition = {
         enhancedProfileNFTAvatar: injectProfileNFTAvatarInTwitter,
         profileAvatar: injectNFTAvatarInTwitter,
         openNFTAvatar: injectOpenNFTAvatarEditProfileButton,
+        tweetNFTAvatar: injectUserNFTAvatarAtTweet,
+        CollectionAvatar: injectNFTContractAtTwitter,
```

Comment:
```suggestion
        collectionAvatar: injectNFTContractAtTwitter,
```

To keep the same code style with others.